tests/components/zwave/test_sensor.py

```diff
@@ -67,7 +69,7 @@ def test_get_device_detects_battery_sensor(mock_openzwave):
 
     device = sensor.get_device(node=node, values=values, node_config={})
     assert isinstance(device, sensor.ZWaveBatterySensor)
-    assert device.device_class == homeassistant.const.DEVICE_CLASS_BATTERY
+    assert device.device_class == BinarySensorDeviceClass.BATTERY
```

Comment:
This is not state attributes, so you should be able to use `is` here

Reply:
And are you sure that it should not be `SensorDeviceClass` here rather than `BinarySensorDeviceClass`

Reply:
I think you are right.  Swapped to SensorDeviceClass.
And agreed, == to is.
Done.